storage/innobase/xtrabackup/test/t/bug1192834-purge.sh

```diff
@@ -0,0 +1,33 @@
+##########################################################################
+# Bug 1192834: Crash during apply with index compaction enabled          #
+##########################################################################
+
+. inc/common.sh
+
+start_server --innodb_file_per_table
+load_dbase_schema sakila
+load_dbase_data sakila
+
+run_cmd $MYSQL $MYSQL_ARGS -e "DELETE FROM payment LIMIT 100" sakila
```

Comment:
I personally find `mysql` shortcut function from `inc/common.sh` somewhat more convenient and less verbose.